### PR TITLE
[1LP][RFR] Add test for appliance console IP configuration

### PR DESCRIPTION
**Pull request by jaryn**
Purpose or Intent
=================

__Adding tests__ for setting the appliance IP address.

Shriver: limiting PRT
{{ pytest: cfme/tests/cli/test_appliance_console.py  -k 'test_appliance_console_static_ipv4 or test_appliance_console_static_ipv6 or test_appliance_console_dhcp' }}

### CODE REVIEW

File: cfme/tests/cli/test_appliance_console.py [hunk 1]
```diff
@@ -374,3 +374,57 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
                 logger.info("{}: no result".format(rule))
         else:
             logger.info("{}: rule not found".format(rule))
+
+
+def test_appliance_console_dhcp(unconfigured_appliance, soft_assert):
+    """'ap' launches appliance_console, '' clears info screen, '1' configure network,
+    '1' configure DHCP, 'y' confirm IPv4 configuration, 'y' IPv6 configuration."""
```
Comment (sshveta) [inline cfme/tests/cli/test_appliance_console.py]: Please make the doc string more readable 
Example : 
   """ Command set - 
        1. 'ap' launches appliance_console, 
        2. ' ' clears info screen,
        3. '1' configure network,
        4. '1' configure DHCP, 
        5. 'y' confirm IPv4 configuration, 
        6. 'y' IPv6 configuration.
   """

File: cfme/tests/cli/test_appliance_console.py [hunk 2]
```diff
@@ -374,3 +374,57 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
...
+    soft_assert(unconfigured_appliance.ssh_client.run_command
+        ("ip a show dev eth0 | grep 'inet6\s.*dynamic'"))
+
+
+def test_appliance_console_static_ipv4(unconfigured_appliance, soft_assert):
+    """'ap' launches appliance_console, '' clears info screen, '1' configure network,
+    '2' configure static IPv4, '' confirm default IPv4 addr, '' confirm default netmask,
+    '' confirm default gateway, '' confirm default primary DNS, '' confirm default secondary DNS,
+    '' confirm default search order, 'y' apply static configuration.
+    """
```
Comment (sshveta) [inline cfme/tests/cli/test_appliance_console.py]: Same here for the doc string 
""" Command set - 
     'ap' launches appliance_console
     '' clears info screen
     '1' configure network
     '2' configure static IPv4
     '' confirm default IPv4 addr
     '' confirm default netmask
     '' confirm default gateway
     '' confirm default primary DNS
     '' confirm default secondary DNS
     '' confirm default search order 
     'y' apply static configuration
"""

File: cfme/tests/cli/test_appliance_console.py [hunk 3]
```diff
@@ -374,3 +374,57 @@ def test_appliance_console_scap(temp_appliance_preconfig, soft_assert):
...
+    soft_assert(unconfigured_appliance.ssh_client.run_command(
+        "ip -4 r show dev eth0 | grep 'default via'"))
+
+
+def test_appliance_console_static_ipv6(unconfigured_appliance, soft_assert):
+    """'ap' launches appliance_console, '' clears info screen, '1' configure network,
+    '3' configure static IPv6, '1::1' set IPv4 addr, '' set deafault prefix length,
+    '1::f' set IPv6 gateway, '' confirm default primary DNS, '' confirm default secondary DNS,
+    '' confirm default search order, 'y' apply static configuration.
+    """
```
Comment (sshveta) [inline cfme/tests/cli/test_appliance_console.py]: Same here for doc string

File: cfme/tests/cli/test_appliance_console.py (hunk 1 above)
Comment (jaryn) [reply cfme/tests/cli/test_appliance_console.py]: I like your suggestion better, but I followed the style of the rest of the file.

**Comment by sshveta** [comment]
Most failures are because of SSH unavailable .
Please comment.

File: cfme/tests/cli/test_appliance_console.py [hunk 4]
```diff
@@ -54,8 +54,12 @@ def test_appliance_console(appliance):
 
 
 def test_appliance_console_set_hostname(configured_appliance):
-    """'ap' launch appliance_console, '' clear info screen, '1' loads network settings, '5' gives
-    access to set hostname, 'hostname' sets new hostname."""
+    """ Commands:
```
Comment (mshriver) [inline cfme/tests/cli/test_appliance_console.py]: :+1: way better formatting, thanks.

**Comment by mshriver** [comment]
I updated PR description in order to limit PRT execution to the new tests only, as all other changes were to docstrings.

Want to see passing tests here, and ensure there's no impact to appliance post-test.

**Pull request merged**